iron-list.html

```diff
@@ -1390,11 +1390,14 @@
 
       forceUpdate = forceUpdate || this._scrollHeight === 0;
       forceUpdate = forceUpdate || this._scrollPosition >= this._estScrollHeight - this._physicalSize;
+      // Probably a bug that style.height is a string like '100px' and it is being compared to
+      // a number?
       forceUpdate = forceUpdate || this.grid && this.$.items.style.height < this._estScrollHeight;
 
       // Amortize height adjustment, so it won't trigger large repaints too often.
       if (forceUpdate || Math.abs(this._estScrollHeight - this._scrollHeight) >= this._optPhysicalSize) {
         this.$.items.style.height = this._estScrollHeight + 'px';
+        this.updateViewportBoundaries();
```

Comment:
Unfortunately the previous repro example in MD Settings has been affected a bit by a follow up CL, so the repro instructions are pretty involved (reverting previous commits, rebuilding Chromium).

I tried to create a minimal reproduction case at http://jsfiddle.net/s6f029j3/12/, which I think is similar to the MD Settings one. I don't understand, why even though iron-list figures out the correct height (see red rectangle in the example), it refuses to render more elements until resize. From reading previous discussions on the matter. Why is the default behavior so surprising (as revealed by numerous discussions on iron-list sizing)? Are you sure that asking devs to call anything after the list is rendered, is the right approach API-wise? Asking especially since this patch seems to fix the issue.